<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>changePwdDialog</class>
    <widget class="QDialog" name="changePwdDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>250</height>
            </rect>
        </property>
        <property name="maximumSize">
            <size>
                <width>400</width>
                <height>250</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>输入新密码</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>50</y>
                    <width>291</width>
                    <height>101</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                    <number>20</number>
                </property>
                <property name="verticalSpacing">
                    <number>20</number>
                </property>
                <item row="0" column="0">
                    <widget class="QLabel" name="userName">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>新密码</string>
                        </property>
                    </widget>
                </item>
                <item row="0" column="1">
                    <widget class="QLineEdit" name="pwd1">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                        </property>
                    </widget>
                </item>
                <item row="1" column="1">
                    <widget class="QLineEdit" name="pwd2">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                        </property>
                    </widget>
                </item>
                <item row="1" column="0">
                    <widget class="QLabel" name="password">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>重复输入</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QPushButton" name="confirmBtn">
            <property name="geometry">
                <rect>
                    <x>160</x>
                    <y>200</y>
                    <width>93</width>
                    <height>28</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>确认</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
